scripts/createEntrypoints.ts

```diff
@@ -13,6 +13,9 @@ createEntryPointFromESMPath('actions', 'actions');
 // Create an Entry Point from everything under bundles/esm/constants -> dist/constants
 createEntryPointFromESMPath('constants', 'constants');
 
+// Create an Entry Point from everything under bundles/esm/params -> dist/params
```

Comment:
Is this related to this pr?

Reply:
Yes it is, it ensures that the Gravity param is importable by the end user

`import Gravity from '@base/params/gravity'